<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>650</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>650</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>650</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Fontys Pick and Place Machine Interface</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>611</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Debug</string>
     </attribute>
     <widget class="QLabel" name="img_fontysLogo">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>370</y>
        <width>331</width>
        <height>171</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>figures/Fontys-Logo.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="openExternalLinks">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>271</width>
        <height>111</height>
       </rect>
      </property>
      <property name="title">
       <string>TCP Connection Settings</string>
      </property>
      <widget class="QLineEdit" name="txt_portNumber">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>40</y>
         <width>110</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>1985</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_22">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>20</y>
         <width>67</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Port</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_connect">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>110</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Connect</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="txt_ipAddress">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>110</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>10.101.10.150</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_23">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>67</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>IP Address</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_disconnect">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>70</y>
         <width>110</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Disconnect</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>10</y>
        <width>201</width>
        <height>211</height>
       </rect>
      </property>
      <property name="title">
       <string>Move X / Y / φ</string>
      </property>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>91</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>X Position</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="txt_xPosition">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>20</y>
         <width>110</width>
         <height>25</height>
        </rect>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
       <property name="readOnly">
        <bool>false</bool>
       </property>
       <property name="placeholderText">
        <string>Input in Millimeters</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>91</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Y Position</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="txt_yPosition">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>50</y>
         <width>110</width>
         <height>25</height>
        </rect>
       </property>
       <property name="placeholderText">
        <string>Input in Millimeters</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_moveCommand">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>180</y>
         <width>110</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Move</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="txt_phiPosition">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>80</y>
         <width>110</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Input in Degrees</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>80</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>φ Position</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="cbx_relativePosition">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>180</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Relative Position</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>120</y>
        <width>261</width>
        <height>51</height>
       </rect>
      </property>
      <property name="title">
       <string>Vacuum Pump</string>
      </property>
      <widget class="QPushButton" name="btn_disableVacuum">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>20</y>
         <width>110</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Disable Vacuum</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_enableVacuum">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>110</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Enable Vacuum</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>170</y>
        <width>261</width>
        <height>51</height>
       </rect>
      </property>
      <property name="title">
       <string>φ Module Light</string>
      </property>
      <widget class="QPushButton" name="btn_enableLight">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>110</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Enable Light</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_disableLight">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>20</y>
         <width>110</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Disable Light</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>10</y>
        <width>261</width>
        <height>111</height>
       </rect>
      </property>
      <property name="title">
       <string>Homing</string>
      </property>
      <widget class="QPushButton" name="btn_homeDevice">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>110</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Home Device</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_homeXAxis">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>20</y>
         <width>110</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Home X Axis</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_homeYAxis">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>110</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Home Y Axis</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_homeZAxis">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>50</y>
         <width>110</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Home Z Axis</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_homePhiAxis">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>80</y>
         <width>110</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Home φ Axis</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>271</width>
        <height>51</height>
       </rect>
      </property>
      <property name="title">
       <string>Miscellaneous</string>
      </property>
      <widget class="QPushButton" name="btn_ping">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>110</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Ping</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_clearDebugConsole">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>20</y>
         <width>110</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Clear Console</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>230</y>
        <width>491</width>
        <height>71</height>
       </rect>
      </property>
      <property name="title">
       <string>Command String</string>
      </property>
      <widget class="QLineEdit" name="txt_commandString">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>341</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="placeholderText">
        <string>Command String</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="cbx_showCommandString">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>181</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>Show Command Before Sending</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_send">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>40</y>
         <width>110</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Send</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>300</y>
        <width>441</width>
        <height>281</height>
       </rect>
      </property>
      <property name="title">
       <string>Console</string>
      </property>
      <widget class="QScrollArea" name="scrollArea_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>431</width>
         <height>251</height>
        </rect>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>429</width>
          <height>249</height>
         </rect>
        </property>
        <widget class="QPlainTextEdit" name="lbl_debugConsole">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>431</width>
           <height>251</height>
          </rect>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </widget>
      </widget>
     </widget>
     <widget class="QLabel" name="lbl_buildVersion">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>560</y>
        <width>421</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="textFormat">
       <enum>Qt::PlainText</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>271</width>
        <height>51</height>
       </rect>
      </property>
      <property name="title">
       <string>Power Motors</string>
      </property>
      <widget class="QPushButton" name="btn_powerOn">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>110</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Enable Motors</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_powerOff">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>20</y>
         <width>110</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Disable Motors</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_11">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>230</y>
        <width>261</width>
        <height>71</height>
       </rect>
      </property>
      <property name="title">
       <string>Pick / Place</string>
      </property>
      <widget class="QPushButton" name="btn_pickComponent">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>110</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Pick Component</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_placeComponent">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>40</y>
         <width>110</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Place Component</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Settings</string>
     </attribute>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>20</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Y offset</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txt_yOffset">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>40</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>X offset</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txt_xOffset">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Batch processing</string>
     </attribute>
     <widget class="QTableView" name="tbl_positionFileView">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>791</width>
        <height>521</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_selectPositionFile">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>530</y>
        <width>81</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Select File</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txt_positionFilePath">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>530</y>
        <width>381</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_sendBatch">
      <property name="geometry">
       <rect>
        <x>710</x>
        <y>530</y>
        <width>81</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Send Batch</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>txt_ipAddress</tabstop>
  <tabstop>txt_portNumber</tabstop>
  <tabstop>btn_disconnect</tabstop>
  <tabstop>btn_connect</tabstop>
  <tabstop>txt_xPosition</tabstop>
  <tabstop>txt_yPosition</tabstop>
  <tabstop>txt_phiPosition</tabstop>
  <tabstop>btn_disableVacuum</tabstop>
  <tabstop>btn_enableVacuum</tabstop>
  <tabstop>btn_disableLight</tabstop>
  <tabstop>btn_enableLight</tabstop>
  <tabstop>btn_clearDebugConsole</tabstop>
  <tabstop>txt_commandString</tabstop>
  <tabstop>scrollArea_2</tabstop>
  <tabstop>lbl_debugConsole</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
